### PR TITLE
Add claimed domains on validator explorer

### PR DESCRIPTION
**Pull request by Pedro-vk**
## Description

_Add claimed domains on validator explorer_

## Tested

_Tested on Chrome_

## Related issues

- Related to #2787

### CODE REVIEW

File: packages/web/src/dev/ValidatorsList.tsx [hunk 1]
```diff
@@ -296,9 +310,9 @@ class ValidatorsList extends React.PureComponent<ValidatorsListProps & I18nProps
               />
             </View>
             {validatorGroups.map((group, i) => (
-              <View key={group.address} onClick={this.expand.bind(this, i)}>
+              <div key={group.id} onClick={this.expand.bind(this, i)}>
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx:321]: why this change to a div?

File: packages/web/src/dev/ValidatorsList.tsx [hunk 2]
```diff
@@ -230,6 +243,7 @@ class ValidatorsList extends React.PureComponent<ValidatorsListProps & I18nProps
     const dir = orderAsc ? 1 : -1
 
     return (data || [])
+      .sort((a: any, b: any) => b.id - a.id)
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx:246]: what is id?

File: packages/web/src/dev/ValidatorsListRow.tsx [hunk 3]
```diff
@@ -40,20 +45,61 @@ export interface CeloGroup {
...
+      event.stopPropagation()
+    }
+    const toggleTooltip = (event: any) => {
+      stopPropagation(event)
+      this.setState({ tooltip: !tooltip })
+    }
     return (
-      <View key={group.address}>
-        <View style={[styles.tableRow, styles.tableRowCont]}>
+      <div style={tooltip ? { zIndex: 2 } : {}}>
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListRow.tsx:101]: why the inline style?

File: packages/web/src/dev/ValidatorsList.tsx (hunk 2 above)
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx]: please type these

File: packages/web/src/dev/ValidatorsList.tsx [hunk 4]
```diff
@@ -49,12 +49,16 @@ interface CeloValidatorGroup {
...
     usd: string
+    claims: any
   }
   accumulatedActive: string
   accumulatedRewards: string
   affiliates: {
     edges: Array<{
       node: {
+        account: {
+          claims: any
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsList.tsx]: please type this

File: packages/web/src/dev/ValidatorsList.tsx (hunk 1 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsList.tsx:321]: Because there was a lot of issues related with the z-index because of View default styling

File: packages/web/src/dev/ValidatorsListRow.tsx (hunk 3 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsListRow.tsx:101]: I moved it with the rest of the styles and crashed the app. Seems styles of a div and `StyleSheet.create` don't work fine.

Comment (aaronmgdr) [reply packages/web/src/dev/ValidatorsListRow.tsx:101]: oh yeah thats true.

File: packages/web/src/dev/ValidatorsListStyles.tsx [hunk 5]
```diff
@@ -87,21 +91,26 @@ export const styles = StyleSheet.create({
     cursor: 'pointer',
     display: 'flex',
     flexDirection: 'row',
-    overflow: 'hidden',
+    // overflow: 'hidden',
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListStyles.tsx:94]: consider deleting line

File: packages/web/src/dev/ValidatorsListStyles.tsx [hunk 6]
```diff
@@ -220,4 +229,46 @@ export const styles = StyleSheet.create({
...
+    textAlign: 'center',
+    marginLeft: 6,
+    position: 'relative',
+  } as any,
+
+  // Tooltip
+  tooltip: {
+    fontWeight: '300',
+    fontSize: 14,
+    backgroundColor: '#585c60',
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListStyles.tsx:250]: nit: would prefer to use color enum here

File: packages/web/src/dev/ValidatorsListRow.tsx [hunk 7]
```diff
@@ -40,20 +45,61 @@ export interface CeloGroup {
...
+interface State {
+  tooltip?: boolean
+}
+
+class ValidatorsListRow extends React.PureComponent<Props & I18nProps, State> {
+  state = {
+    tooltip: false,
+  }
+  tooltipRef = React.createRef<any>()
+  removeDocumentListener: any
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListRow.tsx:65]: this can be typed as `() => void`

**Review by aaronmgdr — APPROVED** [review]
a few small nits but seems good generally

File: packages/web/src/dev/ValidatorsListStyles.tsx (hunk 6 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsListStyles.tsx:250]: It is not a color of the palette

File: packages/web/src/dev/ValidatorsListRow.tsx (hunk 7 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsListRow.tsx:65]: Will fix this things on next PR
